sansa-ml/README.md

```diff
@@ -274,6 +274,49 @@ val resultNodes: Array[Node] = res.as[Node].collect()
 ```
 this sample is taken from a [Scala unit test](https://github.com/SANSA-Stack/SANSA-Stack/tree/develop/sansa-ml/sansa-ml-spark/src/test/scala/net/sansa_stack/ml/spark/utils/SPARQLQueryTest.scala)
 
+### DistAD Distributed Anomaly Detection
```

Comment:
@fmoghaddam  The reference to this subchapter needs to be changed also up int he readme (roughly line 20)

Reply:
fixed